### DISCUSSION TITLE
Snackbar background color not working on iOS 13.3.1.

### DISCUSSION THREAD

**Discussion by cat0363** [CommunityToolkit/Maui#1159] [Q&A]
I tried to show a snackbar with background color set like below, but the 
background color doesn't work on iOS 13.3.1. 

    // Create Snackbar
    var snackbar = Snackbar.Make(
        "This is test message.",
        action: null,
        actionButtonText: string.Empty,
        duration: new TimeSpan(0, 0, 0, 0, 3500),
        visualOptions: new CommunityToolkit.Maui.Core.SnackbarOptions() {
            BackgroundColor = Colors.Red,
            TextColor = Colors.White 
        },
        anchor: null
    );

    // Display Snackbar
    snackbar.Show();

The above code works as intended on Android, but not on iOS 13.3.1.

The build environment and execution environment are as follows.

.NET MAUI CommunityToolkit: 5.1.0
.NET MAUI: 7.0.81
Target Device OS: iOS 13.3.1

In iOS 13.3.1, the background color is displayed as transparent, 
but in iOS 16.3, it is displayed with the specified background color.

[iOS 13.3.1]
![IMG_0258](https://user-images.githubusercontent.com/125236133/233907205-5a98ae67-6875-4582-b16b-bae9bfc3c6e8.PNG)

[iOS 16.3]
![IMG_0336](https://user-images.githubusercontent.com/125236133/233907223-5dc1dfe3-7a8d-456b-845f-e10e9872ac94.PNG)

How can I display a snackbar with a background color on iOS 13.3.1?
Thank you.

I uploaded the source code below.
https://github.com/cat0363/MauiComm-IssueSnackbar.git

**Comment by pictos** [comment]
@cat0363 please open a bug issue, so we can better track it.

**Reply by cat0363** [reply]
Hi, @pictos 
Thank you for your reply.
I have opened this issue as Issue #1160.

**Comment by VladislavAntonyuk** [comment]
Technically we set background color to 
UIStackView: https://github.com/CommunityToolkit/Maui/blob/088ef0c0162b412bd2ad1a54c830fb13310a2565/src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs#L95

possibly color is not converted correctly from MAUI color to ui color

**Reply by cat0363** [reply]
Hi, @VladislavAntonyuk 
Thank you for suggesting the cause of this problem.
You mean there's a difference in color conversion between iOS versions?
